src/dialog/provider.js

```diff
@@ -94,6 +94,13 @@ export default class FlowProvider extends EventEmitter2 {
       fs.writeFileSync(uiPath, JSON.stringify(uiContent, null, 2))
     }
 
+    const flowsDir = path.resolve(this.projectLocation, this.botfile.flowsDir || './flows')
+    fs
+      .readdirSync(flowsDir)
```

Comment:
Flows can be in subdirectories, thus we are using `glob` to search flows (I will fix that)